src/components/SelectCustom/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import DropIcon from "../../assets/icons/arrow_down.png";
+import {
+  ButtonText,
+  DropdownButton,
+  DropdownList,
+  ItemDropDown,
+} from "./styled";
+
+type Props = {
```

Comment:
Nesse caso talvez fique melhor interface.

Mais detallhes: https://blog.logrocket.com/types-vs-interfaces-in-typescript

Reply:
O matheus já resolveu isso e trocou pelo interface